### ISSUE TITLE
Minor issue: Donate/PayPal butto 

### ISSUE THREAD

**Issue by Petclaud** [vitormhenrique/OctoPrint-Enclosure#88]
Tried to make a small donation, but the Donate/PayPal button does not link on gifthub :-)

Maybe the wrong place - sorry - but a kind request for some help: 

I tried to change the TAB order in OctoPrint. The config.yaml file was not present in the folder ~/.OctoPrint hence I created the file via sudo nano config.yaml. In the editor I filled in the following:
appearance:
  components:
    order:
      tab:
      - temperature
      - control
      - gcodeviewer
      - terminal
      - plugin_enclosure
      - timelapse

However, the TAB order is still the same. Any thoughts/advice..?

**Comment by vitormhenrique** [comment]
Hello! thanks for letting me know about the button, I think I fixed...

Are you using an octopi installation or have you installed octoprint from scratch?

**Comment by vitormhenrique** [comment]
btw,  I'm adding a bunch of new features and minor fixes! new release is coming soon...

**Comment by vitormhenrique** [comment]
Hello, did you get the tab order thing worked out?

**Comment by Petclaud** [comment]
Hello. Thx for your feedback. No I did not get the tab order changed via editing the config.yaml file. I do have on one printer an almost 'clean and fresh' Octopi installation and on another one that has been updated already several times (has been running for more than a year). On both of them I could not make it work. However.... a new plugin has arrived (https://plugins.octoprint.org/plugins/taborder/) and that did it for me. Thx for your support!
